addons/account/models/account_account.py

```diff
@@ -1035,33 +1046,33 @@ def _adapt_parent_account_group(self, company=None):
             return
 
         self.flush_model()
-        query = SQL("""
```

Comment:
Always use the `SQL` wrapper.
The preferred way, you can also execute with a simpler API: `records = self.env.execute_query(SQL("..."))`